frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -320,64 +353,119 @@ const AssociatedLandForm: React.FC<IAssociatedLandForm> = (props: IAssociatedLan
     return response?.available;
   };
 
+  const renderPreForm = (): React.ReactNode => {
```

Comment:
This page now displays before the associated land form when passed a building with no associated parcels.